src/routes/profile.js

```diff
@@ -85,7 +102,7 @@ export function profileRoutes() {
       }
 
       ctx.body = 'Profile successfully deleted.';
-    }: Middleware)
```

Comment:
I believe the cast wasn't needed really and this is a bit verbose.

Reply:
I think it was providing the type for the `ctx` variable, otherwise it was an any.

Reply:
Ah right, I'll add the "Context" type in the parameters to get this. Thanks for the feedback!

Reply:
plot twist: the Middleware type here coming from `@koa/router` types the context as `any` :)

Reply:
So, I reused the same typing as you did before, I may have a look at it in a separate patch.